### DISCUSSION TITLE
Why is "use strict" missing?

### DISCUSSION THREAD

**Discussion by codersjj** [babel/babel#15778] [Q&A] [answered]
Last year, I could still see 'use strict':
![image](https://github.com/babel/babel/assets/44868357/8d67034c-3b2f-4d21-a14d-1444967ff648)

But now, I choose the same version(v7.18.13), the compiled code doesn't have 'use strict' at the beginning anymore:
![image](https://github.com/babel/babel/assets/44868357/f51ec99f-e0aa-413f-9d56-2320c6406aa0)

Why?😕

**Comment by nicolo-ribaudo** [comment, marked as answer]
In the second screenshot, your sidebar has "Modules" set to `false`. `false` means "do not compile ESM", so the output is assumed to be ESM that has already implicit strict mode. In the screenshot taken last year, you were probably compiling modules to CommonJS.

**Reply by codersjj** [reply]
Got it! Thanks!❤️
